src/Pages/Cart/Components/CartTotal.js

```diff
@@ -0,0 +1,33 @@
+const CartTotal = ({ cart, total }) => {
+  const discountAmount = 50;
+
+  return (
+    <>
+      <div className="mycart-total-details">
```

Comment:
have a border for this card will look good just suggestion

Reply:
done as you said